src/Pretzel.Logic/Import/BloggerImport.cs

```diff
@@ -102,7 +102,12 @@ private void ImportPost(BloggerPost post)
 
             try
             {
-                fileSystem.File.WriteAllText(Path.Combine(pathToSite, Path.Combine("_posts", fileName)), postContent);
+                var path = Path.Combine(pathToSite, Path.Combine("_posts", fileName));
```

Comment:
We should avoid the call to `Path.GetDirectoryName`:
``` cs
var postsPath = Path.Combine(pathToSite, "_posts");
var path = Path.Combine(postsPath, fileName);

if(!fileSystem.Directory.Exists(postsPath))
...
```

Reply:
Fixed.